### DISCUSSION TITLE
The certificate chain was issued by an authority that is not trusted.

### DISCUSSION THREAD

**Discussion by xiaostoneh** [dataplat/dbatools#9472] [General]
most of commands will report the following error:  "The certificate chain was issued by an authority that is not trusted." what is the solution?

**Comment by dba-richard** [comment]
Hi @xiaostoneh .
A good start is to search for the solution, as this is a very common issue after Microsoft altered how the connection is made to SQL Server. 

I have copied the answer from @wsmelton which you can find in the [issues list](https://github.com/dataplat/dbatools/issues/8796#issuecomment-1458938788):

> 
> You will have to create a custom connection using Connect-DbaInstance to deal with non-standard configurations or connection string settings.
> 
> ```
> $sqlCn = Connect-DbaInstance ... -TrustServerCertificate
> Invoke-DbaQuery -SqlInstance $sqlCn ...
> ```
> 
> A second option that takes affects for all commands is setting it at the configuration level for the module.
> 
> ```
> Get-DbatoolsConfig -FullName sql.connection.trustcert # should see the value set to false
> Set-DbatoolsConfig -FullName sql.connection.trustcert -Value $true
> ```

Hope you found out the issue before this reply was posted, and if not, thanks for being insanely patient.
